backend/src/main/java/com/zpi/backend/reservations/ReservationRepository.java

```diff
@@ -43,5 +45,20 @@ public interface ReservationRepository extends JpaRepository<Reservation, Long>
 
     Page<Reservation>getReservationsByRenter(Pageable pageable, String uuid);
 
+    @Query(
+            value = "SELECT * FROM reservations WHERE date(start_date) = CURRENT_DATE + INTERVAL '2 days' and status_id = 1", nativeQuery = true
+    )
+    List<Reservation> getReservationsStartingInTwoDays();
+    @Query(
+            value = "SELECT * FROM reservations WHERE date(start_date) = CURRENT_DATE and status_id = 1", nativeQuery = true
+    )
```

Comment:
i don't think that hardocoding status id's is a good idea 

![image](https://github.com/zpi-project/gameshare-app/assets/44587062/0bd380eb-2b34-4215-901a-043a1f1fbccd)

as in my db it's different (but it's because of some duplicates that once were in my local table)

Reply:
okay, I'll change that